src/NServiceBus.AcceptanceTests/Routing/SubscriptionBehavior.cs

```diff
@@ -1,9 +1,9 @@
-﻿namespace NServiceBus.AcceptanceTests.PubSub
+﻿namespace NServiceBus.AcceptanceTests.Routing
```

Comment:
I see no particular reason why the acceptance tests should have `NServiceBus.AcceptanceTests` namespace and not `NServiceBus` directly